plugin/src/main/java/com/spotify/plugin/dockerfile/BuildMojo.java

```diff
@@ -93,7 +103,7 @@ public void execute(DockerClient dockerClient)
     }
 
     final String imageId = buildImage(
-        dockerClient, log, verbose, contextDirectory, repository, tag, pullNewerImage, noCache);
+        dockerClient, log, verbose, contextDirectory, repository, tag, pullNewerImage, noCache, buildArgs);
```

Comment:
`buildArgs` is now the seventh argument passed to this method that is an instance variable, perhaps it is time to make `buildImage` non-static so we don't have to have so many parameters passed.